openc3-cosmos-init/plugins/packages/openc3-cosmos-tool-scriptrunner/src/tools/ScriptRunner/ScriptRunner.vue

```diff
@@ -650,22 +650,22 @@ export default {
               divider: true,
             },
             {
-              label: 'Show Environment',
+              label: 'Global Environment',
```

Comment:
Global Environment vs Script Environment. Let me know if you like that terminology.

Reply:
Sounds good.